src/konnektor/tests/network_planners/generators/test_explicit_network_planner.py

```diff
@@ -28,33 +29,42 @@ def test_explicit_network_planner():
     assert get_is_connected(ligand_network)
 
 
-def test_explicit_network_planner_with_indices():
+def test_explicit_network_planner_from_indices():
```

Comment:
Is this test WIP? If so, please add a TODO & link to an issue. If not then maybe we remove it?

Reply:
theyre both added now - sorry for the premature review request (I wasn't sure if we had anything bigger-picture to hash out)